frontend/src/components/layout/Footer.js

```diff
@@ -1,93 +1,87 @@
-import { AppBar, Box, Container, Grid, Typography } from "@mui/material";
-import { getApiUrl, getGithubUrl } from "../../lib/utils";
+import { AppBar, Box, Container, Grid, Typography } from '@mui/material';
+import { getApiUrl, getGithubUrl } from '../../lib/utils';
 
-import Link from "../Link";
+import Link from '../Link';
 
 const Footer = () => {
-  const footerData = [
-    {
-      title: "General",
-      content: [
-        {
-          name: "Home",
-          link: "/",
-        },
-        {
-          name: "About",
-          link: "/about",
-        },
-        {
-          name: "Impressum",
-          link: "https://danielschuster.me/legal-notice",
-        },
-        {
-          name: "Sitemap",
-          link: "/sitemap.xml",
-        },
-      ],
-    },
-    {
-      title: "Database Search",
-      content: [
-        {
-          name: "Characters",
-          link: "/characters",
-        },
-        {
-          name: "Movies",
-          link: "/movies",
-        },
-      ],
-    },
-    {
-      title: "Information",
-      content: [
-        {
-          name: "API",
-          link: getApiUrl(),
-        },
-        {
-          name: "Contribute",
-          link: getGithubUrl(),
-        },
-      ],
-    },
-  ];
+	const footerData = [
+		{
+			title: 'General',
+			content: [
+				{
+					name: 'Home',
+					link: '/',
+				},
+				{
+					name: 'About',
+					link: '/about',
+				},
+				{
+					name: 'Impressum',
+					link: 'https://danielschuster.me/legal-notice',
+				},
+				{
+					name: 'Sitemap',
+					link: '/sitemap.xml',
+				},
+			],
+		},
+		{
+			title: 'Database Search',
+			content: [
+				{
+					name: 'Characters',
+					link: '/characters',
+				},
+				{
+					name: 'Potions',
+					link: '/potions',
+				},
+				{
+					name: 'Movies',
+					link: '/movies',
+				},
```

Comment:
Could you please sort those alphabetically? First Characters, then Movies and finally Potions?